_inc/client/components/forms/styles.scss

```diff
@@ -46,7 +46,7 @@
 		margin-top: 2px;
 	}
 	.jp-form-setting-explanation {
-		color: darken( $gray, 20% );
```

Comment:
There are two similar occurrences in `/_inc/client/components/dash-item/style.scss`. Maybe they should be updated to `$gray-text-min` as well?

```css
.dops-section-header__label {
	color: darken( $gray, 20% );
}

&.is-working,
&.is-premium-inactive {
	.dops-section-header__actions {
		color: lighten( $gray, 20% );
	}
}
```